pkg/testutils/builder/moduletemplate.go

```diff
@@ -34,6 +34,9 @@ func NewModuleTemplateBuilder() ModuleTemplateBuilder {
 			ObjectMeta: apimetav1.ObjectMeta{
 				Name:      random.Name(),
 				Namespace: apimetav1.NamespaceDefault,
+				Labels: map[string]string{
+					shared.IsMandatoryModule: shared.EnableLabelValue,
+				},
```

Comment:
I am a bit confused by this change. Isn't this applying the mandatory module label to all tests?

Reply:
yes but that's for the integration tests in which some don't have the kyma controller initialized. also, there is another followup issue to this to introduce the label using modulectl create command, so anyways the label should be there before the kyma controller

Reply:
ah thanks! I meant to have it only for the mandatory modules integration tests.